doc/design/embargo-mvp.md

```diff
@@ -0,0 +1,141 @@
+# Embargo Minimum Viable Product
+Due to time constraints, we are implementing the core functionality for embargoing dandisets ASAP.
+This document is dedicated to the minimum viable product.
+A subsequent document will describe the full suite of features.
+
+
+# User experience
+
+## Web
+When creating a new dandiset, users will have checkbox to choose whether or not the dandiset should be embargoed.
+If so, the user must also specify an NIH award number (https://era.nih.gov/files/Deciphering_NIH_Application.pdf).
+The award number is an optional argument that is passed to the create dandiset endpoint.
```

Comment:
The award number is optional in terms of the endpoint, but required if a user wishes to create an embargoed dataset. The optionality is tied directly to whether or not the endpoint creates an embargoed or open dandiset, right?

That is, you never have to supply an award number, but you get an embargoed dandiset if and only if you supply one. Is that correct?

Reply:
sounds about right.